hook_test.py

```diff
@@ -0,0 +1,3 @@
+def decode(encoded_message, offset):
```

Comment:
Now that you know the hook works, you probably don't want this around anymore as it doesn't really have anything to do with the ceaser modules.